README.md

```diff
@@ -76,24 +76,11 @@ For the github version make sure you are on the develop branch.
 1. Follow [these instructions](https://github.com/watson-developer-cloud/unity-sdk#configuring-your-service-credentials) to create your Speech To Text and Conversation services and find their credentials (using [IBM Cloud](https://console.ng.bluemix.net/)).
  You can find your workspace ID by selecting the expansion menu on your conversation workspace and selecting `View details`.
     ![View Details Location](doc/source/images/workspace_details.png?raw=true)
-1. Open the script `vr-speech-sandbox-cardboard/SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs`
-1. Fill in the credentials for Speech to Text and Conversation, and the Conversation workspace id:
-```
-    private string stt_username = "";
-    private string stt_password = "";
-    // Change stt_url if different from below
-    private string stt_url = "https://stream.watsonplatform.net/speech-to-text/api";
-
-    private string convo_username = "";
-    private string convo_password = "";
-    // Change convo_url if different from below
-    private string convo_url = "https://gateway.watsonplatform.net/conversation/api";
-    // Change  _conversationVersionDate if different from below
-    private string _conversationVersionDate = "2017-05-26";
-    private string convo_workspaceId = "";
-```
+1. In the Unity Hierarchy view, click on `Player` and then the `Streaming Speech` object.
+1. In the Inspector you will see Variables for `Speech To Text` and `Watson Assistant` and either `CF Authentication` for the Cloud Foundry username and password, or the `IAM Authentication` if you have the IAM apikey. Since you only have only 1 version of these credentials, fill out only one of the two for each service.
+1. Fill out the `Stt_service Url`, the `Asst_service Url`, the `Asst_workspace ID`, and the `Asst_version Date`. There are tool tips which will show help and any defaults.
```

Comment:
Something we don't like to do from a developer experience perspective is use acronyms. We always use the full service names - `Speech to Text` and `Assistant`

Reply:
Done.